docker/redis_tests/walg.env

```diff
@@ -0,0 +1,12 @@
+PGSSLMODE=allow
```

Comment:
Hm, if I understand correctly this variable needed only by pg_tests and overwritten if it is not set in other environment files? In that case, we can specify environment variable in docker-compose file https://docs.docker.com/compose/environment-variables/
It will be like
```
    environment:
      - PGSSLMODE=allow
```
I tested it with removing PGSSLMODE variable from redis_tests - it works.

Reply:
It was a comment related to the whole file :)

Reply:
Oh, I see :). Yeah, sure, we can use common environment file, I implemented it in my PR, it really looks better that way.